### PR TITLE
upper case method

### PR DESCRIPTION
**Pull request by Topru333**

### CODE REVIEW

File: src/main/kotlin/com/exactpro/th2/http/client/util/MessageUtil.kt [hunk 1]
```diff
@@ -67,7 +67,7 @@ private const val HEADER_VALUE_SEPARATOR = ";"
 
 private fun createRequest(head: Message, body: RawMessage): RawHttpRequest {
     val metadata = body.metadata.propertiesMap
-    val method = head.getString(METHOD_FIELD) ?: metadata[METHOD_PROPERTY] ?: DEFAULT_METHOD
+    val method = (head.getString(METHOD_FIELD) ?: metadata[METHOD_PROPERTY])?.uppercase() ?: DEFAULT_METHOD
```
Comment (ivandruzhinin) [inline src/main/kotlin/com/exactpro/th2/http/client/util/MessageUtil.kt:70]: Why do we need this?

**Review by ivandruzhinin — COMMENTED** [review]
Commit message should clearly state what was done

File: src/main/kotlin/com/exactpro/th2/http/client/util/MessageUtil.kt (hunk 1 above)
Comment (Topru333) [reply src/main/kotlin/com/exactpro/th2/http/client/util/MessageUtil.kt:70]: if incoming messages have lowercase method in request, no request are sent by client

Comment (ivandruzhinin) [reply src/main/kotlin/com/exactpro/th2/http/client/util/MessageUtil.kt:70]: > no request are sent by client

Could you explain why?

**Review by ivandruzhinin — APPROVED** [review]
Please also correct commit message

**Pull request merged**